### PR TITLE
Added E861C

### PR DESCRIPTION
**Pull request by SkynetHome**
Added LS Control E861C per Issue #3064

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/database/lscontrol/e861c.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		<Parameter>
+			<Index>1</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Delta Percentage</Label>
+			<Help lang="en">2 % - 56 % -> Delta</Help>
+		</Parameter>
+		<Parameter>
+			<Index>2</Index>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/lscontrol/e861c.xml]: I would make this a list with just the two options.

File: bundles/binding/org.openhab.binding.zwave/database/lscontrol/e861c.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Celsius/Fahrenheit</Label>
+			<Help lang="en">0 - 64 -> Celsius : 64 - 127 -> Fahrenheit</Help>
+		</Parameter>
+		<Parameter>
+			<Index>3</Index>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/lscontrol/e861c.xml]: I would make this a list with just the two options.

**Comment by SkynetHome** [comment]
Its been a while since I looked at this, not sure I understand why I would create a list with two options for parameters that take a range of values.

**Comment by cdjackson** [comment]
Because there's really only 2 values. 0-64 are the same, and 65 to 127 are the same, so just use 2 values - one for C and one for F.

**Pull request closed without merging**